### PR TITLE
Refactor `TestBountyMetrics` To Use A Real Postgres DB For The Test

### PR DESCRIPTION
**Pull request by aliraza556**
## Describe your changes

image

## Issue ticket number and link

Closes: #1792

## Checklist before requesting a review
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by aliraza556** [comment]
Please review this error. I refactored the unit test code according to the requirements, but I am still encountering this error.

image

File: handlers/metrics_test.go [hunk 1]
```diff
@@ -57,51 +131,36 @@ func TestBountyMetrics(t *testing.T) {
...
-			BountiesAssigned:       2,
-			BountiesPaidPercentage: 1,
-			SatsPosted:             1,
-			SatsPaid:               1,
-			SatsPaidPercentage:     1,
-			AveragePaid:            1,
-			AverageCompleted:       1,
-			UniqueHuntersPaid:      1,
-			NewHuntersPaid:         1,
+			BountiesPosted:         4,
```
Comment (elraphty) [inline handlers/metrics_test.go:147]: @aliraza556 SatsPaid, SatsPosted, BountiesPaid, BountieAssigned should not be a static variable

**Review by elraphty — CHANGES_REQUESTED** [review]

**Comment by elraphty** [comment]
> Please review this error. I refactored the unit test code according to the requirements, but I am still encountering this error.
> 
> image

This error occurs if a variable is not present in runtime memory, you have to fins what variable is causing this by debugging.

File: handlers/metrics_test.go (hunk 1 above)
Comment (aliraza556) [reply handlers/metrics_test.go:147]: @elraphty Ok

**Comment by aliraza556** [comment]
Hi @elraphty, Please review this PR

**Pull request merged**